src/python/pants/backend/python/subsystems/setup.py

```diff
@@ -149,6 +149,20 @@ def register_options(cls, register):
                 "We recommend keeping the default of `error` for CI builds."
             ),
         )
+        register(
+            "--run-against-entire-lockfile",
```

Comment:
Thinking out loud..I think this name will work when we have _multiple_ lockfiles. You still are resolving only a single lockfile at a time.

Reply:
Yes, I chose the name to be future-proof with exactly that reading of it in mind.